Screens/PerformanceScreen.js

```diff
@@ -7,14 +7,47 @@ import { connect } from 'react-redux';
 import { loadPerformance } from '../Actions/PerformanceCreator';
 import LocalizeComponent from "../Localization/LocalizedComponent";
 import { BallIndicator } from 'react-native-indicators';
+import ImageLayout from "react-native-image-layout";
 
 class PerformanceScreen extends LocalizeComponent {
 
     componentWillMount() {
```

Comment:
prefer use componentDidMount
Explanation https://daveceddia.com/where-fetch-data-componentwillmount-vs-componentdidmount/

Reply:
I will correct it